A72-open-telemetry-tracing.md

```diff
@@ -41,22 +41,34 @@ tracing instrumentation.
 ### Related Proposals and Documents:
 * [gRFC L29: C++ API Changes for OpenCensus Integration][L29]
 * [gRFC A45: Exposing OpenCensus Metrics and Tracing for gRPC retry][A45]
-* [Microservices observability overview][grpc-observability-public-doc]
 * [gRFC A66: OpenTelemetry Metrics][A66]
 
-## Proposal
-### gRPC OpenTelemetry Tracing API
+# Proposal
+## gRPC OpenTelemetry Tracing API
 We will add tracing functions in grpc-open-telemetry plugin, along with OpenTelemetry
-metrics [gRFC A66][A66]. Languages will keep using gRPC infrastructures, e.g. interceptor and
-stream tracer to implement the feature, the same as Census.
+metrics [gRFC A66][A66]. Internally, the tracing functionality will be implemented
+using existing gRPC infrastructure such as interceptors and stream tracers.
 
-The OpenTelemetry API will coexist with the OpenCensus API. Only one
-`grpc-trace-bin` header will be sent for a single RPC as long as only one of
-OpenTelemetry or OpenCensus is enabled for the channel. The APIs to enable and
-configure OpenTelemetry tracing are different among languages due to different 
-underlying infrastructures.
+#### Propagator Wire Format
```

Comment:
Suggest de-indenting this header 2 levels and moving this section down to right before the migration section.